tests/installation/installer_desktopselection.pm

```diff
@@ -15,7 +15,7 @@ use testapi;
 sub run() {
     my %desktopkeys = (kde => "k", gnome => "g", xfce => "x", lxde => "l", minimalx => "m", textmode => "i");
     assert_screen "desktop-selection", 30;
-    my $d   = get_var("DESKTOP");
+    my $d = check_var('DESKTOP', 'awesome') ? "minimalx" : get_var('DESKTOP');
```

Comment:
this won't scale. We will need the same damn thing for all kind of desktops. Perhaps the awesome test shouldn't use DESKTOP=awesome but DESKTOP=minimalx and some other variable to trigger the test load in main.pm.